src/rich-text-editor.component.js

```diff
@@ -50,6 +50,13 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
     return () => divRef.current.removeEventListener('paste', interceptPaste)
   }, [props.pasteFn])
 
+  useEffect(() => {
+    if (props.onInput) {
```

Comment:
I think this would be easier with just an `onInput` prop on the div.

https://github.com/CanopyTax/bandicoot/pull/89/files#diff-320fb6ae9b4bf4dbf2e4d0e13ca034f3R178

Reply:
Similar to how we're doing onBlur and onFocus:

https://github.com/CanopyTax/bandicoot/blob/d1254d9570f39055d35aa3ff426b5a0145779dac/src/rich-text-editor.component.js#L172